package.json

```diff
@@ -156,6 +158,7 @@
     "@nx/workspace": "17.3.2",
     "@testing-library/react": "13.4.0",
     "@types/cron": "^2.4.0",
+    "@types/hbs": "^4.0.4",
```

Comment:
I moved all dependencies from the scripts folder here so we don't need to add -W flag every time we add new dependency